source/beamline/BioXAS/BioXASBeamline.cpp

```diff
@@ -513,19 +513,6 @@ bool BioXASBeamline::clearGe32Detectors()
 	return true;
 }
 
-void BioXASBeamline::setConnected(bool isConnected)
```

Comment:
`setConnected()` was called by `updateConnected()` slot. And `updateConnected()` was replaced with `onBeamlineComponentConnected()` slot in the `CLSBeamline`